native/SampleApps/SmartSyncExplorer/SmartSyncExplorer/Classes/ContactDetailViewController.m

```diff
@@ -255,20 +255,27 @@ - (void)deleteContact {
     [self.navigationController popViewControllerAnimated:YES];
 }
 
+- (void)undeleteContact {
```

Comment:
Adding undelete support (already in the hybrid and react native versions of the application)